spec/unit/application_spec.rb

```diff
@@ -140,17 +140,28 @@
 
   describe "#namespace" do
 
-    it "should yield a new namespace" do
-      application.namespace :new_namespace do |ns|
-        expect(ns.name).to eq :new_namespace
-      end
+    context "should yield a new namespace" do
+      before { ActiveSupport::Deprecation.behavior = :stderr }
```

Comment:
I guess it should be fine to test breaking changes this way?

Reply:
Use `ActiveSupport::Deprecation.silence do ... end` in test.